Library/ViewModels/DiscoveryEditorialViewModel.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import ReactiveSwift
+
+public typealias DiscoveryEditorialCellValue = (
+  title: String,
+  subtitle: String, imageName: String, tag: String, refTag: RefTag
+)
+
+public protocol DiscoveryEditorialViewModelInputs {
+  func configureWith(_ value: DiscoveryEditorialCellValue)
+  func editorialCellTapped()
+}
+
+public protocol DiscoveryEditorialViewModelOutputs {
+  var imageName: Signal<String, Never> { get }
+  var notifyDelegateViewTapped: Signal<(String, RefTag), Never> { get }
+  var subtitleText: Signal<String, Never> { get }
+  var titleText: Signal<String, Never> { get }
+}
+
+public protocol DiscoveryEditorialViewModelType {
+  var inputs: DiscoveryEditorialViewModelInputs { get }
+  var outputs: DiscoveryEditorialViewModelOutputs { get }
+}
+
+public final class DiscoveryEditorialViewModel: DiscoveryEditorialViewModelType,
+  DiscoveryEditorialViewModelInputs, DiscoveryEditorialViewModelOutputs {
+  public init() {
+    let configureWithValue = self.configureWithValueProperty.signal.skipNil()
+
+    self.imageName = configureWithValue.map { $0.imageName }
```

Comment:
Not using keypaths? No matter, just asking.

Reply:
It's a tuple, so no key path support 😔